pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "semversioner"
+version = "2.0.2"
+description = "Properly manage semver in your repository"
+authors = ["Raul Gomis <raulgomis@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+# The licence and python version classifiers are automatically included by Poetry
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: End Users/Desktop',
+    'Operating System :: OS Independent',
+]
+
```

Comment:
We are missing: 

```
requires-python = ">=3.8"
```

We also need to exclude the `tests` directory in the packaging. Not sure if Poetry does it by default. Any thoughts on that?

Reply:
I just double checked, and the tests are excluded